<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionProperties</class>
 <widget class="QWidget" name="ConnectionProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="choosePage2_rb">
     <property name="text">
      <string>TCP</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="choosePage1_rb">
     <property name="text">
      <string>RTU</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_7">
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0" columnstretch="0" rowminimumheight="0,0,0">
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Адрес сервера</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QLabel" name="label_17">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Укажите адрес сервера modbus (1-247):</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="adress_spbox">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>247</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Параметры RTU соединения</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QFrame" name="frame_13">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>Модуль:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="module_cmbox">
                  <item>
                   <property name="text">
                    <string>CPU</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RSSW в слоте 12</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_14">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_30">
                  <property name="text">
                   <string>Порт:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_31">
                  <property name="text">
                   <string>Скорость:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_32">
                  <property name="text">
                   <string>Биты данных: </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_33">
                  <property name="text">
                   <string>Стоп-биты:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="5">
                 <widget class="QLabel" name="label_34">
                  <property name="text">
                   <string>Чётность:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="7">
                 <widget class="QLabel" name="label_35">
                  <property name="text">
                   <string>Линии:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="ports_cmbox"/>
                </item>
                <item row="1" column="7">
                 <widget class="QCheckBox" name="dtr_chckbox">
                  <property name="text">
                   <string>DTR</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="8">
                 <widget class="QCheckBox" name="rts_chckbox">
                  <property name="text">
                   <string>RTS</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QComboBox" name="stopBits_cmbox">
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1.5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QComboBox" name="dataBits_cmbox">
                  <property name="currentIndex">
                   <number>1</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="baud_cmbox">
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <property name="currentText">
                   <string>9600</string>
                  </property>
                  <property name="currentIndex">
                   <number>2</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>1200</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4800</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19200</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>38400</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>57600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>115200</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="5">
                 <widget class="QComboBox" name="parity_cmbox">
                  <item>
                   <property name="text">
                    <string>NONE</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ODD</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>EVEN</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="6">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Выбор режима:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="6">
                 <widget class="QComboBox" name="mode_cmbox">
                  <item>
                   <property name="text">
                    <string>RS-232</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RS-485</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Временные интервалы протокола</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0" columnstretch="0,0,0">
             <item row="2" column="1">
              <widget class="QSpinBox" name="retry_spbox"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="retry_lbl">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Количество повторных попыток выполнения запросов, шт</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="byteTimeout_lbl">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Максимальный интервал между байтами одного пакета, мс</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="packageTimeout_lbl">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Максимальное время ожидания ответного пакета, мс</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="byteTimeout_spbox">
               <property name="minimum">
                <number>50</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="packageTimeout_spbox">
               <property name="minimum">
                <number>50</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0" columnstretch="0,1" rowminimumheight="0,0">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_13">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Адрес сервера</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="adress_lbl">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Укажите адрес сервера modbus (1-247):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="address_spbox">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>247</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_12">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>Параметры TCP соединения</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_18" columnstretch="0,0,0,0,0">
             <item row="0" column="2">
              <widget class="QLineEdit" name="ip_lnedit">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="inputMask">
                <string>000.000.000.000;_</string>
               </property>
               <property name="text">
                <string>127.000.000.001</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QSpinBox" name="tcpPort_spbox">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="maximum">
                <number>65536</number>
               </property>
               <property name="value">
                <number>502</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="ip_lbl">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>IP:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_47">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Порт:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
